apps/sasquatch/build.gradle

```diff
@@ -88,15 +86,3 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     implementation 'com.android.support.test.espresso:espresso-idling-resource:3.0.2'
 }
-
-apply plugin: 'com.google.gms.google-services'
```

Comment:
I think we also can remove [this]( https://github.com/microsoft/appcenter-sdk-android/blob/develop/apps/sasquatch/google-services.json) file and [these](https://github.com/microsoft/appcenter-sdk-android/blob/develop/apps/sasquatch/src/main/AndroidManifest.xml#L74) lines from manifest.
From maven these lines: [1](https://github.com/microsoft/appcenter-sdk-android/blob/develop/cgmanifest.json#L57) and [2](https://github.com/microsoft/appcenter-sdk-android/blob/develop/cgmanifest.json#L67).
And related constants [here](https://github.com/microsoft/appcenter-sdk-android/blob/117dadf6800466ec9f58fc98d2ffcae31a6bec15/apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java#L70).